io.github.tofudd.lanchat.terminal.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app-id": "io.github.tofudd.lanchat.terminal",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "20.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.dotnet5"
+  ],
+  "finish-args": [
+    "--share=network",
+    "--env=DOTNET_ROOT=/app/dotnet/lib",
+    "--filesystem=home"
+  ],
+  "command": "Lanchat",
+  "modules": [
+    {
+      "name": "Lanchat",
+      "buildsystem": "simple",
+      "build-options": {
+        "arch": {
+          "aarch64": {
+            "env" : {
+              "RUNTIME": "linux-arm64"
+            }
+          },
+          "arm": {
+            "env" : {
+              "RUNTIME": "linux-arm"
+            }
+          },
+          "x86_64": {
+            "env" : {
+              "RUNTIME": "linux-x64"
+            }
+          }
+        }
+      },
+      "build-commands": [
+        "/usr/lib/sdk/dotnet5/bin/dotnet publish -c Release --source ../nuget-sources Lanchat.Terminal/Lanchat.Terminal.csproj --runtime $RUNTIME --self-contained true",
+        "cp -r --remove-destination /run/build/Lanchat/Lanchat.Terminal/bin/Release/net5.0/$RUNTIME/publish/ /app/bin/",
+        "install -D io.github.tofudd.lanchat.terminal.svg /app/share/icons/hicolor/128x128/apps/io.github.tofudd.lanchat.terminal.svg",
+        "install -D io.github.tofudd.lanchat.terminal.desktop /app/share/applications/io.github.tofudd.lanchat.terminal.desktop",
+        "install -D io.github.tofudd.lanchat.terminal.appdata.xml /app/share/metainfo/io.github.tofudd.lanchat.terminal.appdata.xml"
+      ],
+      "sources": [
+      {
+          "type": "file",
+          "path": "io.github.tofudd.lanchat.terminal.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html